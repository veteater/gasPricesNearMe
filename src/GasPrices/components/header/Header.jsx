import React from 'react';
import LocationControls from '../locationControls/LocationControls';

const Header = () => {
  return (
    <header>
      <h1>Gas Prices Near Me</h1>
      <LocationControls />
    </header>
  );
};

export default Header;