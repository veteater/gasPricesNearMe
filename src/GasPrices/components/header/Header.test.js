import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  it('renders without crashing', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
