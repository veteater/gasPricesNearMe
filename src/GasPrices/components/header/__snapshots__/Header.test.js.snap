// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<header>
  <h1>
    Gas Prices Near Me
  </h1>
  <LocationControls />
</header>
`;
